### PR TITLE
[SYCL][ESIMD][E2E] Fix aot_mixed.cpp

### PR DESCRIPTION
**Pull request by sarnex**
Arg was wrong. Manually tested this. CI is current broken because of this,

### CODE REVIEW

File: sycl/test-e2e/ESIMD/aot_mixed.cpp [hunk 1]
```diff
@@ -7,9 +7,9 @@
 //===----------------------------------------------------------------------===//
 // TODO: Enable on other GPUs once internal ticket is fixed
 // REQUIRES: ocloc && gpu-intel-gen12
-// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen -Xsycl-target-backend=spir64_gen -Xs "-device tgllp" -o %t.sycl.out -DENABLE_SYCL=0 %s
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen -Xsycl-target-backend=spir64_gen "-device tgllp" -o %t.sycl.out -DENABLE_SYCL=0 %s
```
Comment (v-klochkov) [inline sycl/test-e2e/ESIMD/aot_mixed.cpp:10]: That is odd. In my notes that `-Xs` is required. Seems something has changed in driver, right?

Comment (sarnex) [reply sycl/test-e2e/ESIMD/aot_mixed.cpp:10]: Nope, it's because we have `-Xsycl-target-backend` which is kind of similar to `-Xs` but lets you provide backend options for a specific target because we support compiling for multiple targets at once. `-Xs` will apply to all targets (I think), here it doesn't matter because there's only one.

If you want to use `-Xs` only, drop `-Xsycl-target-backend=spir64_gen "-device tgllp"` and use `-Xs "-device tgllp"

TLDR it's just the way the `RUN` command was originally written and I missed it.

**Review by v-klochkov — APPROVED** [review]
